Amplitude/Amplitude.m

```diff
@@ -257,6 +277,15 @@ - (void)initializeApiKey:(NSString*) apiKey
 
 - (void)initializeApiKey:(NSString*) apiKey userId:(NSString*) userId
 {
+    UIApplicationState state = [UIApplication sharedApplication].applicationState;
+    [self initializeApiKey:apiKey userId:userId startSession:(state != UIApplicationStateBackground)];
+}
```

Comment:
@curtisliu, I'm particularly interested in what you think of this part of the change. By default, it will start a session if the app is opened in the foreground and not start a session if the app is loaded by a push notif. Devs will get this behavior if they don't change their code. If they want other behavior, they'd have to use the initializeApiKey method with :startSession.